.github/workflows/e2e-suite.yml

```diff
@@ -36,6 +36,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run the integration test suite
-        run: make testint
+        run: |
+          timestamp=$(date +'%Y%m%d%H%M')
+          report_filename="${timestamp}_cli_test_report.xml"
+          
+          if pytest -m tests/integration --disable-warnings --junitxml="${report_filename}"; then
+            linode-cli obj --cluster us-southeast-1 put "${report_filename}" test-report-bucket
+          else
+            echo "Tests failed, but attempting to upload results anyway"
+            linode-cli obj --cluster us-southeast-1 put "${report_filename}" test-report-bucket
+            exit 1
+          fi
         env:
           LINODE_CLI_TOKEN: ${{ secrets.LINODE_TOKEN }}
+          LINODE_CLI_OBJ_ACCESS_KEY: ${{ secrets.LINODE_CLI_OBJ_ACCESS_KEY }}
+          LINODE_CLI_OBJ_SECRET_KEY: ${{ secrets.LINODE_CLI_OBJ_SECRET_KEY }}
```

Comment:
I wonder if we could dynamically generate these using `linode-cli obj regenerate-keys` to skip adding them as GitHub secrets.